__tests__/api.test.js

```diff
@@ -90,6 +90,53 @@ describe('GET /api/articles/:article_id', () => {
     })
 })
 
+describe('GET /api/articles/:article_id/comments', () => {
+    test('when queried with an article id, Responds with status code 200 and with response object with correct properties', () => {
+        return request(app)
+        .get('/api/articles/1/comments')
+        .expect(200)
+        .then(({ body }) => {
+            const comments = body.comments
+            comments.forEach((comment) => {
+                expect(comment).toMatchObject({ 
+                    article_id: 1,
+                    comment_id: expect.any(Number),
+                    votes: expect.any(Number),
+                    author: expect.any(String),
+                    body: expect.any(String),
+                    created_at: expect.any(String),
+                })
+            })          
+        });
+    })
+    test('the most recent comments are served first ', () => {
+        return request(app)
+        .get('/api/articles/1/comments')
+        .expect(200)
+        .then(({ body }) => {
+            const comments = body.comments
+            console.log(comments)
+            expect(comments).toBeSortedBy('created_at', {descending: true})
+        })
```

Comment:
Please add another test for a:
Status 200, valid ID, but has no comments responds with an empty array of comments. You will need to add the functionality for this too 👀

Reply:
test added